app/assets/javascripts/discourse/app/components/legacy-header-icon-shim.gjs

```diff
@@ -0,0 +1,25 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { schedule } from "@ember/runloop";
+import { PANEL_WRAPPER_ID } from "discourse/widgets/header";
+import PanelWrapper from "./glimmer-header/panel-wrapper";
+
+export default class LegacyHeaderIconShim extends Component {
+  @tracked panelElement;
+
+  constructor() {
+    super(...arguments);
+    schedule("afterRender", () => {
+      this.panelElement = document.querySelector(`#${PANEL_WRAPPER_ID}`);
+    });
```

Comment:
Noting for other reviewers - this is obviously not an ideal pattern, but this is only being used by the old widget-based header. @janzenisaac and I agreed that it's safer/easier than trying to figure out how to pass the element around in widget state 😅